src/NextFederationPlugin.js

```diff
@@ -465,6 +465,13 @@ class NextFederationPlugin {
    * @returns {void}
    */
   apply(compiler) {
+    if (!this._extraOptions.disableMFClient) {
```

Comment:
make this option be something like

`automaticPageStitching` or something that's easy to understand what the option does.

Reply:
agreed
still thinking of a better name

`automaticPageStitching` is also a bit strange

Reply:
Why about "aggregateRemoteRoutes" ?

Reply:
I think that `mf_client` should be enabled by default. So to disable it, it will be better to start the option from `disable...`. But it's true if we publish v6.

If we want to publish `mf_clent` under v5, in such a case your name is good.

Anyway, this option will completely disable mf_client & react hooks `useMFClient`, `useMFRemote`. `aggregateRemoteRoutes` will not assume that hooks stop working. That's why I called it `disabeMFClient`

Reply:
I'd start disabled just so we can get feedback from users beforehand. Otherwise it may break some use case. 

So we document it in readme and can get user feedback. If no bug reports are filed, we can flip it to enable by default with next major version. Could also probably time it with SSR release too

Reply:
Perhaps we can release this in `v6`?? along with NX + Typescript migration.. I'm already seeing few breaking changes with Migration.

Reply:
@ScriptedAlchemy yep let's gather feedback first
renamed it to `aggregateRemoteRoutes` in https://github.com/module-federation/nextjs-mf/pull/103/commits/89faaca37a914afb74b40e58ec2f29371f2160e1

Reply:
@pavandv suppose your turn in 10-16 hours 😉
But it looks like you'll need to open a new branch and cherry-pick your changes to a new branch from scratch.

<img width="384" alt="Screen Shot 2022-09-15 at 13 36 14" src="https://user-images.githubusercontent.com/1946920/190343816-a894b45a-b6cf-4034-87c0-6b6a203e7f27.png">

Reply:
Yeah I want to keep us on v5 for right now. Keep features a flags we enable. Then v6 we can release with SSR + auto page